src/sentry/api/endpoints/group_details.py

```diff
@@ -211,9 +210,7 @@ def get(self, request, group):
         if last_release:
             last_release = self._get_release_info(request, group, last_release)
 
-        tags = list(GroupTagKey.objects.filter(
-            group_id=group.id,
-        )[:100])
+        tags = tagstore.get_group_tag_keys(group.id)[:100]
```

Comment:
I know this doesn't do a `LIMIT` anymore, but other endpoints (see below) return them all. We also return all `TagKey`s in other places (a `GroupTagKey` must have a matching `TagKey`) so I think the `LIMIT` was added out of habit.

I *could* add `limit` kwargs to every method if we wanted. I didn't see it used anywhere else yet.